src/main/java/com/exasol/dbcleaner/CleanRequestBuilder.java

```diff
@@ -0,0 +1,35 @@
+package com.exasol.dbcleaner;
+
+public class CleanRequestBuilder {
+    private boolean cleanObjects = true;
```

Comment:
A list of enumerations would be more change-friendly here.

Reply:
Yes, but the contract is harder to define. This is why I opted for a DTO.